scripts/benchmark.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+# require success for commands
+set -e
+
+# Parse command line arguments
+MOCK_BALANCES=${MOCK_BALANCES:-true}
+
+# Use python3 as default, but fall back to python if python3 doesn't exist
+PYTHON_CMD=python3
+if ! command -v $PYTHON_CMD &> /dev/null
+then
+    PYTHON_CMD=python
+fi
+
+# set key name
+keyname=admin
+
+# clean up old sei directory
+rm -rf ~/.sei
+echo "Building..."
+# install seid with benchmark support (includes mock_balances)
+echo "Building with benchmark and mock balances support enabled..."
+make install-bench
+# initialize chain with chain ID and add the first key
+~/go/bin/seid init demo --chain-id sei-chain
+~/go/bin/seid keys add $keyname --keyring-backend test
+# add the key as a genesis account with massive balances of several different tokens
+~/go/bin/seid add-genesis-account $(~/go/bin/seid keys show $keyname -a --keyring-backend test) 100000000000000000000usei,100000000000000000000uusdc,100000000000000000000uatom --keyring-backend test
+# gentx for account
+~/go/bin/seid gentx $keyname 7000000000000000usei --chain-id sei-chain --keyring-backend test
+# add validator information to genesis file
+KEY=$(jq '.pub_key' ~/.sei/config/priv_validator_key.json -c)
+jq '.validators = [{}]' ~/.sei/config/genesis.json > ~/.sei/config/tmp_genesis.json
+jq '.validators[0] += {"power":"7000000000"}' ~/.sei/config/tmp_genesis.json > ~/.sei/config/tmp_genesis_2.json
+jq '.validators[0] += {"pub_key":'$KEY'}' ~/.sei/config/tmp_genesis_2.json > ~/.sei/config/tmp_genesis_3.json
+mv ~/.sei/config/tmp_genesis_3.json ~/.sei/config/genesis.json && rm ~/.sei/config/tmp_genesis.json && rm ~/.sei/config/tmp_genesis_2.json
+
+echo "Creating Accounts"
+# create 10 test accounts + fund them
+python3  loadtest/scripts/populate_genesis_accounts.py 20 loc
+
+~/go/bin/seid collect-gentxs
+# update some params in genesis file for easier use of the chain localls (make gov props faster)
+cat ~/.sei/config/genesis.json | jq '.app_state["gov"]["deposit_params"]["max_deposit_period"]="60s"' > ~/.sei/config/tmp_genesis.json && mv ~/.sei/config/tmp_genesis.json ~/.sei/config/genesis.json
```

Comment:
masih has added in QoL improvements for both config and genesis modifications via seictl:
https://github.com/sei-protocol/seictl

we should probably use that instead for simplicity (eg. check for util installation at the beginning of script, and if not, recommend user to do that)